core/src/main/java/org/apache/accumulo/core/client/admin/compaction/CompactionConfigurer.java

```diff
@@ -34,7 +36,7 @@ public interface CompactionConfigurer {
   /**
    * @since 2.1.0
    */
-  public interface InitParameters {
+  interface InitParameters {
```

Comment:
This class is public API. If a user has their own implementation, does removing `public` here cause some type of runtime exception?

Reply:
I do not think so as it is still public since its inside an interface.  My IDE highlighted that these would be public w/o the keyword and that it was unnecessary so I removed them.